arm-trafficmanager/2017-05-01/swagger/trafficmanager.json

```diff
@@ -254,24 +278,29 @@
           }
         ],
         "responses": {
-          "200": { 
-             "description": "The Traffic Manager Endpoint was deleted successfully.",
-             "schema": { 
-               "$ref": "#/definitions/DeleteOperationResult" 
-             } 
-          },
-          "204": { 
-             "description": "The Traffic Manager Endpoint does not exist. It could have been deleted on a previous request.",
-             "schema": { 
-               "$ref": "#/definitions/DeleteOperationResult" 
-             } 
+          "200": {
+            "description": "The Traffic Manager Endpoint was deleted successfully.",
+            "schema": {
+              "$ref": "#/definitions/DeleteOperationResult"
+            }
+          },
+          "204": {
+            "description": "The Traffic Manager Endpoint does not exist. It could have been deleted on a previous request.",
+            "schema": {
+              "$ref": "#/definitions/DeleteOperationResult"
```

Comment:
This looks like an incorrectly modeled response, no? Having a 204 returning data seems incorrect to me...